### PR TITLE
revert conditional expression to CALL, fix joern#91

### PR DESCRIPTION
**Pull request by bbrehm**
Fix for [joern#91](https://github.com/ShiftLeftSecurity/joern/issues/91).

### CODE REVIEW

File: src/test/scala/io/shiftleft/fuzzyc2cpg/astnew/AstToCpgTests.scala [hunk 1]
```diff
@@ -419,9 +419,11 @@ class AstToCpgTests extends WordSpec with Matchers {
...
       block.checkForSingle()
       val call = block.expandAst(NodeTypes.CALL)
-      val condition = call.expandAst(NodeTypes.CONTROL_STRUCTURE)
-      condition.check(1, _.value2(NodeKeys.CODE), expectations = "(foo == 1) ? bar : 0")
-      condition.check(1, _.value2(NodeKeys.PARSER_TYPE_NAME), expectations = "ConditionalExpression")
+      val conditionalExpr = call.expandAst(NodeTypes.CALL)//formerly control structure
+      conditionalExpr.check(1, _.value2(NodeKeys.CODE), expectations = "(foo == 1) ? bar : 0")
+      conditionalExpr.check(1, _.value2(NodeKeys.NAME), expectations = "<operator>.conditionalExpression")
+      val params = conditionalExpr.expandAst()
+      List(params(0)).check(1, _.value2(NodeKeys.CODE), expectations = "foo == 1")
```
Comment (ml86) [inline src/test/scala/io/shiftleft/fuzzyc2cpg/astnew/AstToCpgTests.scala]: Wrong style. Please check like other properties.

File: src/test/scala/io/shiftleft/fuzzyc2cpg/astnew/AstToCpgTests.scala [hunk 2]
```diff
@@ -456,6 +458,7 @@ class AstToCpgTests extends WordSpec with Matchers {
 
       val forBlock = forLoop.expandAst(NodeTypes.BLOCK).filterOrder(4)
       forBlock.checkForSingle()
+      Operators.and
```
Comment (ml86) [inline src/test/scala/io/shiftleft/fuzzyc2cpg/astnew/AstToCpgTests.scala]: Why is this here?

Comment (bbrehm) [reply src/test/scala/io/shiftleft/fuzzyc2cpg/astnew/AstToCpgTests.scala]: I failed at intellij.
